Configuration/AlCa/python/autoCond.py

```diff
@@ -2,33 +2,33 @@
 
     ### NEW KEYS ###
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Run1
-    'run1_design'       :   '76X_mcRun1_design_v10',
+    'run1_design'       :   '80X_mcRun1_design_v0',
     # GlobalTag for MC production (pp collisions) with realistic alignment and calibrations for Run1
-    'run1_mc'           :   '76X_mcRun1_realistic_v10',
+    'run1_mc'           :   '80X_mcRun1_realistic_v0',
     # GlobalTag for MC production (Heavy Ions collisions) with realistic alignment and calibrations for Run1
-    'run1_mc_hi'        :   '76X_mcRun1_HeavyIon_v10',
+    'run1_mc_hi'        :   '80X_mcRun1_HeavyIon_v0',
     # GlobalTag for MC production (p-Pb collisions) with realistic alignment and calibrations for Run1
-    'run1_mc_pa'        :   '76X_mcRun1_pA_v10',
+    'run1_mc_pa'        :   '80X_mcRun1_pA_v0',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Run2
-    'run2_design'       :   '76X_mcRun2_design_v13',
+    'run2_design'       :   '80X_mcRun2_design_v0',
     # GlobalTag for MC production with pessimistic alignment and calibrations for Run2
-    'run2_mc_50ns'      :   '76X_mcRun2_startup_v12',
+    'run2_mc_50ns'      :   '80X_mcRun2_startup_v0',
     #GlobalTag for MC production with optimistic alignment and calibrations for Run2
-    'run2_mc'           :   '76X_mcRun2_asymptotic_v13',
+    'run2_mc'           :   '80X_mcRun2_asymptotic_v0',
     # GlobalTag for MC production (Heavy Ions collisions) with optimistic alignment and calibrations for Run2
-    'run2_mc_hi'        :   '76X_mcRun2_HeavyIon_v12',
+    'run2_mc_hi'        :   '80X_mcRun2_HeavyIon_v0',
     # GlobalTag for Run1 data reprocessing
-    'run1_data'         :   '80X_dataRun1_v0',
+    'run1_data'         :   '80X_dataRun2_v1',
     # GlobalTag for Run2 data reprocessing
-    'run2_data'         :   '80X_dataRun2_v0',
+    'run2_data'         :   '80X_dataRun2_v1',
     # GlobalTag for Run1 HLT: it points to the online GT
-    'run1_hlt'          :   '76X_dataRun1_HLT_frozen_v10',
+    'run1_hlt'          :   '80X_dataRun2_HLT_frozen_v0',
     # GlobalTag for Run2 HLT: it points to the online GT
-    'run2_hlt'          :   '76X_dataRun2_HLT_frozen_v10',
+    'run2_hlt'          :   '76X_dataRun2_HLT_frozen_v0',
```

Comment:
@diguida I guess you meant here `80X_dataRun2_HLT_frozen_v0` instead.... the diff looks good [CondDB browers](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/80X_dataRun2_HLT_frozen_v0/76X_dataRun2_HLT_frozen_v11)